docs/usage.rst

```diff
@@ -17,6 +17,10 @@ sends Logstash messages using UDP and TCP. For example:
   test_logger.setLevel(logging.INFO)
   test_logger.addHandler(AsynchronousLogstashHandler(
       host, port, database_path='logstash.db')))
+
+  # If you don't want to write to a SQLite database, then you do
+  # not have to specify a database_path.
+  # NOTE: Messages are lost between process restarts.
```

Comment:
Maybe add a few more words to the NOTE like "without a SQLite database", just to make clear this NOTE refers to the case with the in-memory database.